.github/workflows/static-checks.yml

```diff
@@ -25,17 +25,13 @@ jobs:
             - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
 
             - name: Use desired version of NodeJS
-              uses: actions/setup-node@38d90ce44d5275ad62cc48384b3d8a58c500bb5f # v2.2.2
+              uses: actions/setup-node@v2
               with:
                   node-version: 14
                   cache: npm
 
             - name: Npm install
-              # A "full" install is executed, since `npm ci` does not always exit
-              # with an error status code if the lock file is inaccurate.
-              #
-              # See: https://github.com/WordPress/gutenberg/issues/16157
```

Comment:
Given the time it passed since that issue was created, I'm assuming that the issue is fixed (we were still in travis) but we'll see.